frontend/app/src/components/home/admin/ReviewTranslationCorrections.svelte

```diff
@@ -9,11 +9,12 @@
     import HoverIcon from "../../HoverIcon.svelte";
     import Menu from "../../Menu.svelte";
     import MenuItem from "../../MenuItem.svelte";
-    import type {
-        CandidateTranslations,
-        OpenChat,
-        RejectReason,
-        TranslationCorrection,
+    import Refresh from "svelte-material-icons/Refresh.svelte";
+    import {
```

Comment:
This seems worse than how it was before.

Reply:
I didn't manually do that...

Reply:
Ah I know why that happened - I'll fix it